medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+
+### Unmapped fields
+
+The following Medication Statement fields are not currently populated by the adaptor:
+- meta.versionId
+- meta.lastUpdated
+- extension(ChangeSummary)
+- partOf
+- category
+- derivedFrom
+- reasonNotTaken
+- reasonCode
+- reasonReference
+- taken <sup>1</sup>
+
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+  "resource": {
+    "resourceType": "MedicationStatement",
+    "id": "AEDDB53B-A30B-4EC9-A20F-A059C29A1C3E-MS",
+    "meta": {
+      "profile": [
+        "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-MedicationStatement-1"
+      ]
+    },
+    "extension": [
+      {
+        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescribingAgency-1",
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "system": "https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescribingAgency-1",
+              "code": "prescribed-at-gp-practice",
+              "display": "Prescribed at GP practice"
+            }
+          ]
+        }
+      }
+    ],
+    "identifier": [
+      {
+        "system": "https://PSSAdaptor/B83002",
+        "value": "AEDDB53B-A30B-4EC9-A20F-A059C29A1C3E-MS"
+      }
+    ],
+    "basedOn": [
+      {
+        "reference": "MedicationRequest/AEDDB53B-A30B-4EC9-A20F-A059C29A1C3E"
+      }
+    ],
+    "status": "completed",
+    "medicationReference": {
+      "reference": "Medication/9fa0d37d-912f-44c2-855b-06d1e8623c97"
+    },
+    "effectivePeriod": {
+      "start": "2010-02-10",
+      "end": "2010-02-10"
+    },
+    "dateAsserted": "2010-02-10T08:31:19+00:00",
+    "subject": {
+      "reference": "Patient/c782c5ef-8235-4acf-945b-268a1bbe9d64"
+    },
+    "taken": "unk",
+    "dosage": [
+      {
+        "text": "To Be Used As Directed"
+      }
+    ]
+  }
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+A XML HL7 `MedicationStatement` is mapped from a FHIR `medicationRequest`.  This can be one of two types, `ORDER` or `PLAN` which is determined by value of `medicationRequest.intent`
+
+### Order
+
+For details of order mapping see the JSON FHIR > XML HL7 mapping for [Medication](../medications/README.md)
+
+### Plan
+
+| Mapped to (XML HL7)                                                                                                | Mapped from (JSON FHIR / other source )                                                                                            |
+|--------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| MedicationStatement \[@classCode] \[@moodCode]                                                                     | fixed value `@classCode="SBADM"`, fixed value `@moodCode="INT"`                                                                    |
```

Comment:
I don't think we've used this syntax to combine two atttributes elsewhere, which may lead to confusion.